overlord/snapstate/snapmgr.go

```diff
@@ -178,10 +179,45 @@ const (
 	NotBlocked
 )
 
+// BlobsSideInfo contains the side information for a snap and related components
+// installed in the system.
+type BlobsSideInfo struct {
```

Comment:
why not define this the same way as the *Marshal below? also maybe this could be called just SideInfo?